GitUI/Editor/FileViewer.cs

```diff
@@ -600,7 +600,9 @@ private string GetFileText(string fileName)
 
             if (File.Exists(path))
             {
-                using (var reader = new StreamReader(path, Module.FilesEncoding))
+                // StreamReader disposes of 'fileStream'.
+                var fileStream = File.Open(path, FileMode.Open, FileAccess.Read, FileShare.ReadWrite);
```

Comment:
Can you dispose `fileStream` as in https://msdn.microsoft.com/library/ms182334.aspx ?